model/user/db.go

```diff
@@ -570,3 +570,18 @@ func UpsertOneFromExisting(ctx context.Context, oldUsr *DBUser, newEmail string)
 
 	return newUsr, errors.Wrapf(err, "unable to insert new user '%s'", newUsername)
 }
+
+func FindByEmail(ctx context.Context, email string) (*DBUser, error) {
```

Comment:
Please add a function comment, and move this function up to where the other FindBy functions are.